api/v1/instancemanager.go

```diff
@@ -26,6 +28,15 @@ type GCPInstance struct {
 	MinCPUPlatform string `json:"min_cpu_platform"`
 }
 
+type DockerInstance struct {
```

Comment:
these properties are not Docker specific, they might be relevant to all hosts. 

Let's address populating these properties in a different PR and keep this commit focused on ownership only.

Reply:
Done